services/ux-backend/handlers/onboardingtokens/handler.go

```diff
@@ -52,3 +58,40 @@ func handleUnsupportedMethod(w http.ResponseWriter, r *http.Request) {
 		klog.Errorf("failed write data to response writer: %v", err)
 	}
 }
+
+func getAllocatedQuotaInGib(r *http.Request) (*int, error) {
+	var quota = struct {
+		StorageQuota int    `json:"storageQuota"`
```

Comment:
Using the word quota inside an object that is already defined as quota creates confusion. 
A quota is a quantity that is built from a unit and a value
```suggestion
		Value int    `json:"value"`
```